webpack.config.js

```diff
@@ -0,0 +1,55 @@
+var path = require('path');
+var NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+
+module.exports = {
+    target: ['web', 'es5'],
+    entry: './lib/index.js',
+    output: {
+        path: path.resolve('./build'),
+        filename: 'plotly.js',
+        library: {
+            name: 'Plotly',
+            type: 'umd',
+            umdNamedDefine: true
+        }
+    },
+    module: {
+        rules: [{
+            test: /\.js$/,
+            exclude: /node_modules/,
+            use: {
+                loader: 'babel-loader'
+            }
+        }, {
+            test: /\.js$/,
+            include: /node_modules[\\\/](buffer|pngjs)[\\\/]/,
+            use: {
+                loader: 'babel-loader',
+                options: {
+                    babelrc: false,
+                    configFile: false,
+                    presets: [
+                        '@babel/preset-env'
+                    ]
+                }
+            },
+        }]
+    },
+    resolve: {
+        fallback: {
+            'stream': require.resolve('stream-browserify'),
+            'buffer': require.resolve('buffer/'),
+            'assert': require.resolve('assert/')
+        }
+    },
+    plugins: [
+        new NodePolyfillPlugin({ includeAliases: ['process'] })
```

Comment:
Did you figure out what this is needed for? Is it something in our code or in a dependency?